app-privacy-policy.md

```diff
@@ -0,0 +1,177 @@
+description: Privacy policy for the Minetest application.
+small: |
+  Last updated: 2023-05-10
+  (<a href="https://github.com/minetest/minetest.github.io/commits/master/app-privacy-policy.md">View updates</a>)
+layout: page_subtitle
+---
+
+<style>
+	h2 {
+		margin-top: 5rem !important;
+	}
+</style>
+
+Minetest may connect to the following services during its operation:
+
+* The main website (www.minetest.net): used to get the most recent version
+* The server list: used to load the server list in "Play Online"
+* ContentDB: used to power the Browse Online Content dialog
+* Game servers: third-party servers when playing online
+
+Note: this policy only applies to the Minetest application, accessing the
+mentioned services directly using a web browser is not covered.
+
+### Table of contents
+
+- [Version checking](#version-checking)
+	- [Information collected](#information-collected)
+	- [How it is used](#how-it-is-used)
+- [Loading server list](#loading-server-list)
+	- [Information collected](#information-collected-1)
+	- [How it is used](#how-it-is-used-1)
+	- [Who has access](#who-has-access)
+	- [Period of retention](#period-of-retention)
+	- [Location](#location)
+- [Browsing Online Content (ContentDB)](#browsing-online-content-contentdb)
+	- [Information collected](#information-collected-2)
+	- [How it is used](#how-it-is-used-2)
+	- [Who has access](#who-has-access-1)
+	- [Period of retention](#period-of-retention-1)
+	- [Location](#location-1)
+	- [Removal requests](#removal-requests)
+- [Online play](#online-play)
+- [Third parties](#third-parties)
+- [Future changes to privacy policy](#future-changes-to-privacy-policy)
+
+
+
+## Version checking
+
+### Information collected
+
+When you open Minetest, it may contact www.minetest.net to fetch information
+about the most recent version. This HTTP request will contain the following
+information:
+
+* IP address
+* Minetest version
+* Platform, OS, device, etc
+
+### How it is used
+
+The version information is hosted on GitHub pages, we do not log requests.
+
+
+## Loading server list
+
+### Information collected
+
+When you open the Play Online tab, Minetest will request the server list. HTTP
+requests are logged with the following information:
+
+* IP address
+* Minetest version
+* Platform, OS, device, etc
+
+### How it is used
+
+Logged HTTP requests are used to aid with debugging, development, and user
+support. They are also used to generate aggregated user statistics, see
+<https://kitsunemimi.pw/tmp/>.
+
+### Who has access
+
+sfan5 runs the server list and has access to the logs. The logs may be shared
+with others to aid in debugging, but care will be taken to remove any personal
+information.
+
+Aggregated statistics are shared publicly.
+
+### Period of retention
+
+Logged HTTP requests are stored for 6 months.
+
+### Location
+
+The server list is currently located in the Netherlands.
+By using this service, you give permission for the data to be moved as needed.
```

Comment:
the data the server list displays is a different matter but we're still talking about http logs with client requests, right?
there is no need ever to move the *logs* to anywhere else

Reply:
You may change the country the server is hosted in